packages/dds/tree/tsconfig.json

```diff
@@ -1,3 +1,19 @@
 {
-	"extends": "./tsconfig.esnext.json",
+	"extends": "@fluidframework/build-common/ts-common-config.json",
```

Comment:
Why not continue using Tyler's defns?
```
	"extends": [
		"../../build/build-common/tsconfig.base.json",
		"../../build/build-common/tsconfig.esm.json",
	],
```
Things I like:
* Less stuff in local tsconfig (node16 and composite=true are already set)
* No publishing packages to pick up base tsconfig changes.

Two files isn't ideal, but we can flatten .base.json and .esm.json in a second pass.

Reply:
tree has never been using those. I think changing the tsconfig to use the common pattern is good. I am just not going to combine two different things.